pkg/controller/perconaservermongodb/mgo.go

```diff
@@ -453,7 +453,31 @@ func (r *ReconcilePerconaServerMongoDB) createSystemUsers(cr *api.PerconaServerM
 		return errors.Wrap(err, "failed to get cluster admin")
 	}
 
-	err = mongo.CreateUser(context.Background(), cli, monitorUser.Username, monitorUser.Password, roleClusterMonitor)
+	err = mongo.CreateRole(context.Background(), cli, "explainRole",
```

Comment:
we need to create these roles only for 1.12.0

Reply:
fixed 8142bdc4dfadcd3ae81da64b374c4b15b783e51b